src/Trigger/RabbitMQTriggerAttributeBindingProvider.cs

```diff
@@ -42,7 +49,16 @@ public Task<ITriggerBinding> TryCreateAsync(TriggerBindingProviderContext contex
                 return Task.FromResult<ITriggerBinding>(null);
             }
 
-            string connectionString = Resolve(attribute.ConnectionStringSetting) ?? _options.Value.ConnectionString;
+            string connectionString = _options.Value.ConnectionString;
+

```

Comment:
move this to Utility. Default to options only if connectionString  after the catch